.gitignore

```diff
@@ -2,3 +2,4 @@ vendor/
 composer.lock
 phpunit.xml
 .env
+.idea
```

Comment:
I'd prefer user-specific items to be ignored in a global gitignore file instead of the project-specific one

Reply:
But this is my only problem with the PR :)

Reply:
Removed it from .gitignore